docs/update.md

```diff
@@ -0,0 +1,29 @@
+# Update kube-prometheus
+
+You may wish to fetch changes made on this project so they are available to you.
+
+## Update jb
+
+`jb` may have been updated so it's a good idea to get the latest version of this binary:
+
+```shell
+$ go get -u github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+```
+
+## Update kube-prometheus
+
+The command below will sync with upstream project:
+
+```shell
+$ jb update
+```
+
+## Compile the manifests and apply
+
+Once updated, just follow the instructions under "Compiling" and "Apply the kube-prometheus stack" to apply the changes to your cluster.
```

Comment:
Could "compiling" and "apply the kube-prometheus stack" have a link to the appropriate place?

Reply:
Fixed.